gatsby-site/src/components/Subscribe/subscribe.css

```diff
@@ -0,0 +1,85 @@
+:root {
+    --colarkfontcolor: #0111FC;
+    --colarkfont: 'Mukta Malar', sans-serif;
+    --background: #FFFFFF;
+}
+
+.subscription-container {
+    text-align: right;
+    width: 100%;
+    font-family: var(--colarkfont);
+}
+
+.form-control {
+    display: inline-block;
+    margin: 0 auto;
+    border: 5px solid white;
+    color: rgba(0, 0, 0, 0.25);
+    width: 400px;
+    border-radius: 0px;
+}
+
+.subscription-header {
+    text-align: left;
+    font-size: 24px;
+    color:white;
+    margin-left: 250px;
+}
+
+.subscription-input {
+    width: 500px;
+    text-align: center;
+    border: 3px solid white;
+    margin: 0 auto;
+    font-weight: 200px;
+    box-shadow: 5px 10px 18px rgba(0, 0, 0, .5);
+    background: white;
+    border-radius: 5px;
+}
+
+.subscribesubmit {
```

Comment:
This is a "code style" comment, but generally a class name style definition should be of the format `.subscribe_submit`, `.subscribe-submit`, or `.subscribeSubmit`. Additionally, it is preferred that the casing is the same across a project. I'm personally less concerned about having them all be the same, but for readability, one of the above options is useful I think!